### ISSUE TITLE
Enhancement: provide advice to the user about good extents for FP 

### ISSUE THREAD

**Issue by severinmenard** [fieldpapers/fieldpapers#147]
There is such advice with the Walking-Papers:

![selection_020](https://cloud.githubusercontent.com/assets/1291055/8072241/f2519920-0f02-11e5-8fca-1265064dea0d.png)

**Comment by rbracket** [comment]
Yes, excellent idea.  For documenting intersection level detail for pedestrian features and some bicycle services--this would save much time.  Currently I resize the bounding box, workaround some random rescaling. start over.....
